File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/AcceptSpeakerInvite.java
package me.grishka.houseclub.api.methods;

import me.grishka.houseclub.api.BaseResponse;
import me.grishka.houseclub.api.ClubhouseAPIRequest;

public class AcceptSpeakerInvite extends ClubhouseAPIRequest<BaseResponse> {
    public AcceptSpeakerInvite(String channel, int userID) {
        super("POST", "accept_speaker_invite", BaseResponse.class);
        requestBody = new Body(channel, userID);
    }

    private static class Body {
        public String channel;
        public int userId;

        public Body(String channel, int userId) {
            this.channel = channel;
            this.userId = userId;
        }
    }
}

File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/AcceptSpeakerInvite.kt
package me.grishka.houseclub.api.methods

import me.grishka.houseclub.api.BaseResponse
import me.grishka.houseclub.api.ClubhouseAPIRequest

class AcceptSpeakerInvite(channel: String, userID: Int) :
    ClubhouseAPIRequest<BaseResponse?>("POST", "accept_speaker_invite", BaseResponse::class.java) {
    private class Body(var channel: String, var userId: Int)

    init {
        requestBody = Body(channel, userID)
    }
}